sdk/spring/azure-spring-boot-test-selenium-common/src/main/java/com/azure/test/aad/common/AADSeleniumITHelper.java

```diff
@@ -1,18 +1,16 @@
-package com.azure.test.aad.selenium;
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.test.aad.common;
```

Comment:
Why  change package name?

Reply:
This class is the common helper for AAD selenium in the selenium-common project, it's in the same location with `SeleniumITHelper`, you mean it should keep the package name 'com.azure.test.aad.selenium.AADSeleniumITHelper', or change to `com.azure.test.aad.common.selenium.AADSeleniumITHelper`?